src/ec/packages/ec-component-datepicker/datepicker.story.js

```diff
@@ -1,35 +1,66 @@
 /* eslint-disable no-param-reassign */
 import { storiesOf } from '@storybook/html';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
-import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+import { withKnobs, text, boolean, select } from '@storybook/addon-knobs';
 import withCode from '@ecl-twig/storybook-addon-code';
+import { getExtraKnobs, buttonLabels } from '@ecl-twig/story-utils';
 
 import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
-import data from './demo/data';
+import dataDefault from './demo/data';
 import datepicker from './ecl-datepicker.html.twig';
 import notes from './README.md';
 
+const prepareDatePicker = data => {
+  data.label = text('label', data.label, buttonLabels.required);
+  data.icons_path = select(
+    'icons_path',
+    [defaultSprite],
+    defaultSprite,
+    buttonLabels.required
+  );
+  data.helper_text = text(
+    'helper_text',
+    data.helper_text,
+    buttonLabels.required
+  );
+  data.invalid = boolean('invalid', false, buttonLabels.cases);
+  data.invalid_text = text(
+    'invalid_text',
+    data.invalid_text,
+    buttonLabels.optional
+  );
+  data.disabled = boolean('disabled', false, buttonLabels.cases);
+  data.required = boolean('required', data.required, buttonLabels.cases);
+  data.required_text = text(
+    'required_text',
+    data.required_text,
+    buttonLabels.optional
+  );
+  data.optional_text = text(
+    'optional_text',
+    data.optional_text,
+    buttonLabels.optional
+  );
+  data.placeholder = text(
```

Comment:
i think for now you can move this into the optional parameters, as we know there is no documentation page to base this on, but yeah, it looks more like an "option" than a requirement

Reply:
You haven't changed this, but since there is no official docs about what is required and what is not for this component we can leave this as required.